src/service/ui/src/controllers/basket.ts

```diff
@@ -0,0 +1,18 @@
+import {Request, Response} from 'express'
+import * as template from 'lib/ui/template'
+import * as learningRecord from './learning-record'
+
+export let basketPage = async (req: Request, res: Response) => {
```

Comment:
For future, make toplevel a function declaration as opposed to an arrow function.